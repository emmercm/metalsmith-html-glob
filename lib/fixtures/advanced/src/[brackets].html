<!DOCTYPE html>
<html lang="en">
<body>
[Brackets]
</body>
</html>
