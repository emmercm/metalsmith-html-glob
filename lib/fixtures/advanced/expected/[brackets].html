<!DOCTYPE html><html lang="en"><head></head><body>
[Brackets]


</body></html>